packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -41,11 +58,13 @@ export default abstract class BaseAction {
 	}
 
 	protected async runSuiteCloudCommand(args: { [key: string]: string } = {} ) {
+        this.logToOutputExecutionDetails();
```

Comment:
Identation seems different with following line. I think it's because there are parts of the file with tabs and others with spaces

Reply:
Updated the file. Replaced all tabs with spaces.

Reply:
It should be all tabs instead of spaces according to the .pretierrc file in the root of our netsuite-suitecloud-sdk repo.
```
{
	"tabWidth": 4,
	"trailingComma": "es5",
	"singleQuote": true,
	"useTabs": true,
	"printWidth": 150
}
```

Reply:
Updated to make use of tabs.